internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/storagenode/StorageNodeReadTest.java

```diff
@@ -340,8 +340,7 @@ private HttpResponse sendHeartbeatRequest(CloseableHttpAsyncClient client, Strin
     StringBuilder sb = new StringBuilder().append("http://")
         .append(serverAddress)
         .append("/")
-        .append(QueryAction.HEALTH.toString().toLowerCase())
-        .append("?f=b64");
```

Comment:
Can we really remove this param? Maybe it doesn't matter if it's a test... but this affects how the request param is encoded.

Reply:
IIUC the format is used to encode keys and in this test its a HEATH query request, which is just a command without anything else.

Reply:
Oh yeah, right. Makes sense.